tests/functional/provisioning/test_dev_path_mapping_file.py

```diff
@@ -792,3 +793,91 @@ def test_dev_path_mapping_heketi_node_delete(self):
             use_percent, use_percent_after,
             "Failed to execute IO's in the app pod {} after respin".format(
                 pod_name))
+
+    @pytest.mark.tier2
+    @podcmd.GlustoPod()
+    def test_dev_path_volume_expansion(self):
+        """Validate dev path mapping for file volumes expansion"""
+
+        pvc_size, pvc_amount, expand_pvc_size = 1, 2, 2
+        vol_names, pvc_names = [], []
+
+        # Create PVC's
+        sc_name = self.create_storage_class(allow_volume_expansion=True)
+
+        pvc_names = self.create_and_wait_for_pvcs(
+            pvc_size=pvc_size, pvc_amount= pvc_amount, sc_name=sc_name)
```

Comment:
Change `pvc_amount= pvc_amount`  according to pep8

Reply:
done